test/Microsoft.AspNetCore.Mvc.IntegrationTests/ActionParametersIntegrationTest.cs

```diff
@@ -400,60 +400,57 @@ private class Person5
             });
             var modelState = testContext.ModelState;
 
-            // Act
-            var result = await argumentBinder.BindModelAsync(parameter, testContext);
-
-            // Assert
-            Assert.True(result.IsModelSet);
-            Assert.True(modelState.IsValid);
-            var model = Assert.IsType<Class1>(result.Model);
-            Assert.Null(model.Property1);
-            var keyValuePair = Assert.Single(modelState);
-            Assert.Equal("Name", keyValuePair.Key);
-            Assert.Equal("James", keyValuePair.Value.AttemptedValue);
-            Assert.Equal("James", keyValuePair.Value.RawValue);
+            // Act & Assert
+            var exception = await Assert.ThrowsAsync<InvalidOperationException>(() => argumentBinder.BindModelAsync(parameter, testContext));
+            Assert.Equal(
+                string.Format(
+                    "Could not create an instance of type '{0}' as it's either an abstract type or a struct or does not have a parameterless constructor.",
+                    typeof(ClassWithNoDefaultConstructor).FullName),
+                exception.Message);
         }
 
         [Fact]
-        public async Task ActionParameter_BindingToStructModel_Fails()
+        public async Task ActionParameter_BindingToStructModel_WithNoParameterlessConstructor_DoesNotCreateModel()
```

Comment:
Leave `_WithNoParameterlessConstructor` out of the name; all value types _have_ a parameterless constructor.